README.md

```diff
@@ -626,9 +627,184 @@ class BookList extends React.Component {
 
 # Child nodes
 
-> TODO
-> ViewChild vs refs
-> ContentChild vs this.props.children
+### AngularJS
+In an [AngularJS component](https://docs.angularjs.org/guide/component), we have access to the child node by injecting [`$element`](https://docs.angularjs.org/api/ng/function/angular.element) to its controller.
+```js
+class TextInputCtrl {
+  constructor($scope, $element) {
+    // $element is passed to the constructor and stored in an instance variable.
+    this.$scope = $scope;
+    this.$element = $element;
+  }
+  
+  $postLink() {
+    // When the links are done, we can use the $element attribute.
+    // It has a jqLite wrapped instance of the DOM element,
+    // so we have access to some jQuery methods.
+    const input = this.$element.find('input');
+    input.on('change', console.log);
+  }
+}
+
+const component = {
+  controller: TextInputCtrl
+  template: `
+    <div>
+      <input type="text" />
+    </div>
+  `
+};
+
+angular.module('app', [])
+  .component('textInput', component);
+```
+
+### Angular
+Angular provides two ways to deal with child nodes: `ViewChild(ren)` and `ContentChild(ren)`. They both have the same purpose, but there are different use cases for them.
+
+#### ViewChild
+[`ViewChild`](https://angular.io/api/core/ViewChild) returns the specified element from the view DOM. [`ViewChildren`](https://angular.io/api/core/ViewChildren) does the same, but for multiple elements (returning as a `QueryList`).
+```ts
+import {
+  Component,
+  Directive,
+  ViewChild,
+  AfterViewInit
+} from '@angular/core';
+
+@Directive({ selector: 'child' })
+export class Child {}
+
+@Component({
+  selector: 'parent',
+  template: '<child></child>'
+})
+export class Parent implements AfterViewInit {
+  // We use the @ViewChild decorator to get the <child> directive
+  @ViewChild(Child) viewChild: Child;
+  
+  ngAfterViewInit() {
+    // The element is only avaliable when the ngAfterViewInit lifecycle hook is reached.
+    console.log(this.viewChild);
+  }
+}
+```
+
+#### ContentChild
+`ViewChild` **don't** include elements that exists within the `ng-content` tag. So we have [`ContentChild`](https://angular.io/api/core/ContentChild), that **only** includes elements that exists within the `<ng-content>` tag. [`ContentChildren`](https://angular.io/api/core/ContentChildren) does the same, but for multiple elements (also returning a `QueryList`).
+
```

Comment:
I think the description may not be clear enough for beginners. 
Take a look here:
https://stackoverflow.com/a/34327754

Reply:
Oh, I agree, it's really a bit confusing. I read the SO question, and I'll think in a better way to explain that for beginners and let you know when it's finished.

Reply:
Sure, let me know when you're ready. The rest looks good :+1: 
Don't forget to fix the conflicts.

Reply:
Hey, how is it going? 😄

Reply:
Hello! Sorry for the delay. :sweat_smile:
But I think it's finished! Can you take a look? Ask me if you need more changes.

Reply:
Cool! I'll take a look 💪